node/internal/fs/streams_test.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+import { createWriteStream, WriteStream } from "./streams.ts";
+import { assertEquals } from "../../../testing/asserts.ts";
+
+Deno.test({
```

Comment:
Same comment in this file - is it possible to leverage the Node test suite more?

Reply:
As with the other comment, duplicate tests have been reorganized in https://github.com/denoland/deno_std/pull/2634/commits/67266461986bf0938bb03c18a5a777e1ab010b49 and https://github.com/denoland/deno_std/pull/2634/commits/a7a852cdcaa8adc9a34312ea3c8df7e2e752f464.
Regarding the Node test suite, please see the separate comment added below.